infra/terraform/test-org/org/folders.tf

```diff
@@ -32,7 +33,7 @@ module "folders-ci" {
   source  = "terraform-google-modules/folders/google"
   version = "~> 1.0"
 
-  parent_id   = replace(local.folders["ci"], "folders/", "")
+  parent_id   = replace(local.folders["ci-projects"], "folders/", "")
```

Comment:
This change requires a change to https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/terraform/test-org/org/locals.tf#L24 in order to be valid.

Reply:
`ci-shared` should also be added to the local.

Reply:
Added to locals